chia/full_node/mempool_manager.py

```diff
@@ -553,11 +553,17 @@ def get_spendbundle(self, bundle_hash: bytes32) -> Optional[SpendBundle]:
             return self.mempool.spends[bundle_hash].spend_bundle
         return None
 
-    def get_mempool_item(self, bundle_hash: bytes32) -> Optional[MempoolItem]:
-        """Returns a MempoolItem if it's inside one the mempools"""
-        if bundle_hash in self.mempool.spends:
-            return self.mempool.spends[bundle_hash]
-        return None
+    def get_mempool_item(self, bundle_hash: bytes32, include_pending: bool = False) -> Optional[MempoolItem]:
+        """
+        Returns a MempoolItem if it's inside one the mempools.
```

Comment:
I can't parse this sentence. is it supposed to say "inside one *of* the mempools"?
But we only have one mempool, so I still wouldn't know what that means really. presumably "it" refers to the `bundle_hash` we pass in here, right?

Reply:
I've spawned https://github.com/Chia-Network/chia-blockchain/issues/13984 for future cleanup to resolve this convo